dimagi/commcare-hq
# Tile config UI

## Product Description

Add the option to configure a case list tile without using a template

<img width="970" alt="image" src="https://github.com/dimagi/commcare-hq/assets/1946138/4b411f3c-c763-4e12-98f7-a0c8001ac949">


## Technical Summary

https://dimagi-dev.atlassian.net/browse/USH-3486

## Feature Flag

* REC/USH: Case tile templates
* USH: Configure custom case list tile

## Safety Assurance

### Safety story

New ui is behind a feature flag

### Automated test coverage

* Test for generating suite.xml

### QA Plan

* "Manual Configure Case Tiles" Option only shows up when "USH: Configure custom case list tile" feature flag is enabled
* Preview is shown for custom configured tiles
* Custom configuration gets saved and rendered in the suite.xml
* Case list shows tiles according to configuration
* Fields with "search only" and "address" formats do not show up in tiles

### Rollback instructions




### Labels & Review
